selection.go

```diff
@@ -241,7 +241,7 @@ func (s *controlledSelector) HandleSuccessResponse(m *stun.Message, local, remot
 	s.log.Tracef("Found valid candidate pair: %s", p)
 	if p.nominateOnBindingSuccess {
 		if selectedPair := s.agent.getSelectedPair(); selectedPair == nil ||
-			(selectedPair != p && selectedPair.priority() <= p.priority()) {
+			(selectedPair != p && ((!s.agent.enableUseCandidateCheckPriority && s.agent.lite) || selectedPair.priority() <= p.priority())) {
```

Comment:
how about make it a agent method `needsToCheckPriorityOnNominated`

Reply:
Good call @cnderrauber . Updated with a helper function.